Modelica/Electrical/Analog/Examples/OpAmps/Differentiator.mo

```diff
@@ -42,7 +41,6 @@ equation
     annotation (Line(points={{10,-10},{40,-10}}, color={0,0,255}));
   annotation (Documentation(info="<html>
 <p>This is a (inverting) differentiating amplifier. Resistance R can be chosen, capacitance C is defined by the desired time constant resp. frequency.</p>
-<p>Note: <code>vOut</code> measure the negative output voltage.</p>
```

Comment:
I noticed this comment has been erased for all of the examples, even though the component remains.
I assume it is intentional and because it is self-explanatory, right?

Reply:
Yes I changed the polarity of measurement of the output voltage, therefore removed the line in the documentation. 
I saw that it is much easier to understand: This is an inverting circuit, and I see this fact looking at the output voltage without thinking about the polarity of the voltage sensor.